target/linux/at91/image/sama5.mk

```diff
@@ -54,23 +54,26 @@ endef
 
 define Device/at91-sama5d2_xplained
   $(Device/evaluation-dtb)
-  DEVICE_TITLE := Microchip(Atmel AT91) SAMA5D2 Xplained
+  DEVICE_VENDOR := Microchip
+  DEVICE_MODEL := SAMA5D2 Xplained
   KERNEL_SIZE := 6144k
   $(Device/evaluation-sdimage)
 endef
 TARGET_DEVICES += at91-sama5d2_xplained
 
 define Device/at91-sama5d27_som1_ek
   $(Device/evaluation-dtb)
-  DEVICE_TITLE := Microchip(Atmel AT91) SAMA5D27 SOM1 Ek
+  DEVICE_VENDOR := Microchip
+  DEVICE_MODEL := SAMA5D27 SOM1 Ek
   KERNEL_SIZE := 6144k
   $(Device/evaluation-sdimage)
 endef
 TARGET_DEVICES += at91-sama5d27_som1_ek
 
 define Device/at91-sama5d2_ptc_ek
   $(Device/evaluation-dtb)
-  DEVICE_TITLE := Microchip(Atmel AT91) SAMA5D2 PTC Ek
+  DEVICE_VENDOR := Microchip
+  DEVICE_MODEL := SAMA5D2 PTC EK
```

Comment:
Upper-case EK by intention? Didn't do that for the previous device ...

Reply:
it's a typo